### PR TITLE
Move monitoring demo to the learning transport

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: tutorials/monitoring-demo/walkthrough-1.md [hunk 1]
```diff
@@ -1,47 +1,45 @@
...
+You can see that the Sales endpoint has the highest processing time, so it is taking the longest to process messages. At low volumes of traffic, it's difficult to see the impact that this has on the rest of the system. Let's add some more traffic.
 
 **Find the ClientUI endpoint window and toggle *High-Throughput* mode. Now go back to the ServicePulse Monitoring Tab.**
 
-In high throughput mode, the ClientUI endpoint sends 20 orders to Sales every second. Now look at the processing time and throughput of each endpoint. The throughput of Sales will increase and then plateau at around 3 messages / second. This is as fast Sales can currently process messages. It can process 4 messages in parallel but each message takes 1.3 seconds to process.
+In high throughput mode, the ClientUI endpoint sends 20 orders to Sales every second. Now, look at the processing time and throughput of each endpoint. The throughput of Sales will increase and then plateau at around three messages / second. This is as fast Sales can currently process messages. It can process four messages in parallel, but each message takes 1.3 seconds to process.
```
Comment (tmasternak) [inline tutorials/monitoring-demo/walkthrough-1.md]: Same question why not 3 and 4?

File: tutorials/monitoring-demo/walkthrough-3.md [hunk 2]
```diff
@@ -25,18 +25,18 @@ Processing time is the time it takes for the endpoint to process a single messag
...
 **Run the sample solution. Open ServicePulse to the Monitoring tab.**
 
 ![Service Pulse monitoring tab showing sample endpoints](servicepulse-monitoring-tab-sample-low-throughput.png)
 
-NServiceBus endpoints frequently rely on other resources to do their work. This might take the form of a database server that holds persisted data or a web server that hosts an API that the endpoint needs to call. The endpoints themselves are designed to tolerate failure but there are some early indicators that failure is coming.
+NServiceBus endpoints frequently rely on other resources to do their work. This might take the form of a database server that holds persisted data or a web server that hosts an API that the endpoint needs to call. The endpoints themselves are designed to tolerate failure, but there are some early indicators that failure is coming.
 
 
-### Processing messages is getting slower
+### Processing messages are getting slower
```
Comment (tmasternak) [inline tutorials/monitoring-demo/walkthrough-3.md]: This should stay as was I think. "Is" in here means the processing and the messages AFIU

File: tutorials/monitoring-demo/walkthrough-3.md [hunk 3]
```diff
@@ -25,18 +25,18 @@ Processing time is the time it takes for the endpoint to process a single messag
...
 
-The first indication that an endpoint is going to run into trouble is when processing messages starts to slow down. This is indicated by an increase in processing time. This means that database queries and web api calls are taking longer to process than they were before.
+The first indication that an endpoint is going to run into trouble is when processing messages start to slow down. This is indicated by an increase in processing time. This means that database queries and web API calls are taking longer to process than they were before.
```
Comment (tmasternak) [inline tutorials/monitoring-demo/walkthrough-3.md]: "Starts" should stay as it previously was I think.

File: tutorials/monitoring-demo/walkthrough-setup.md [hunk 4]
```diff
@@ -26,29 +25,15 @@ After unblocking the zip file, extract its contents into a folder. For the rest
...
 Open the `MonitoringDemo` folder and double-click on `run.bat`. This script will:
 
-1. Ask you to select a SQL Server to use as a transport. All of the components of the demo communicate by using a shared SQL Server database. You will be prompted to *use an existing SQL Server instance* or have the demo create it's *own dedicated SQL LocalDB instance*.
-2. You will be prompted to run the remaining steps with elevated privileges. This is needed so that processes can bind to their network ports and for SQL database creation.
-2. Create a LocalDB instance if required.
-2. Create the SQL database catalog with all of the necessary tables that the demo components will use.
-3. Update the configuration files for all of the components with the correct connection string details.
-4. Run ServiceControl components, each in their own window.
+1. You will be prompted to run the remaining steps with elevated privileges. This is needed so that processes can bind to their network ports.
+2. Update the configuration files for all of the components with the correct connection string details.
```
Comment (tmasternak) [inline tutorials/monitoring-demo/walkthrough-setup.md]: Step two is not executed anymore. The LT folders are relative and hardcoded right?

File: tutorials/monitoring-demo/index.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
-reviewed: 2018-03-15
+reviewed: 2018-11-27
 summary: A self-contained demo solution that you can run to explore the monitoring features of the Particular Service Platform.
 suppressRelated: true
 redirects:
 - tutorials/monitoring/demo
 ---
 
-The best way to get started with the monitoring features in the Particular Service Platform is to try them out on ServicePulse with a real system. This downloadable sample contains all of the necessary parts of the platform, already configured and ready to run. It also includes 4 sample endpoints that all communicate by exchanging messages over the SQL Transport.
+The best way to get started with the monitoring features in the Particular Service Platform is to try them out on ServicePulse with a real system. This downloadable sample contains all of the necessary parts of the platform, already configured and ready to run. It also includes 4 sample endpoints that all communicate by exchanging messages over the Learning Transport.
```
[the remark this answers is not in the document]
Comment (danielmarbach) [reply tutorials/monitoring-demo/index.md]: I'm good with that

File: tutorials/monitoring-demo/walkthrough-1.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (danielmarbach) [reply tutorials/monitoring-demo/walkthrough-1.md]: https://www.dailywritingtips.com/10-rules-for-writing-numbers-and-numerals/

File: tutorials/monitoring-demo/walkthrough-1.md [hunk 6]
```diff
@@ -51,53 +49,53 @@ Let's see what the impact of optimizing the sales endpoint is.
...
 
 The original endpoint will also get a boost in productivity as it will process the remaining message types faster on average. This results in the endpoint throughput going up.
 
 #### Increasing maximum concurrency
 
-Alternatively, you can increase the maximum concurrency level and allow for a higher degree of parallel processing. The default maximum concurrency level is the number of cores of the host machine. Increasing this to 2, 3, 4 times or even more can increase throughput if your handlers are waiting a lot on remote calls to complete.
+Alternatively, you can increase the maximum concurrency level and allow for a higher degree of parallel processing. The default maximum concurrency level is the number of cores of the host machine. Increasing this to 2, 3, four times or even more can increase throughput if your handlers are waiting a lot on remote calls to complete.
```
[the remark this answers is not in the document]
Comment (danielmarbach) [reply tutorials/monitoring-demo/walkthrough-1.md]: See above

[the remark this answers is not in the document]
Comment (danielmarbach) [reply tutorials/monitoring-demo/walkthrough-1.md]: but for consistency reason I need to spell out two and three as well

File: tutorials/monitoring-demo/walkthrough-1.md (hunk 1 above)
Comment (danielmarbach) [reply tutorials/monitoring-demo/walkthrough-1.md]: See above

File: tutorials/monitoring-demo/walkthrough-3.md (hunk 2 above)
Comment (danielmarbach) [reply tutorials/monitoring-demo/walkthrough-3.md]: You are right. Will change back

File: tutorials/monitoring-demo/walkthrough-3.md (hunk 3 above)
Comment (danielmarbach) [reply tutorials/monitoring-demo/walkthrough-3.md]: Correct. Good catch

File: tutorials/monitoring-demo/walkthrough-setup.md (hunk 4 above)
Comment (danielmarbach) [reply tutorials/monitoring-demo/walkthrough-setup.md]: You are right. Good catch!

File: tutorials/monitoring-demo/walkthrough-1.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tmasternak) [reply tutorials/monitoring-demo/walkthrough-1.md]: Good read. Thx! My guess was that we what to have numerals e.g. 4 to make it easier for clients to switch between text and what they see on the dashboard. That being said I'm fine with following the rules.

**Comment by danielmarbach** [comment]
@tmasternak applied your feedback

**Pull request merged**